src/server/database/Updater/UpdateFetcher.cpp

```diff
@@ -4,19 +4,17 @@
  */
 
 #include "UpdateFetcher.h"
-#include "Common.h"
 #include "CryptoHash.h"
 #include "DBUpdater.h"
 #include "Field.h"
 #include "Log.h"
 #include "QueryResult.h"
 #include "Tokenize.h"
 #include "Util.h"
-#include <boost/filesystem/operations.hpp>
-#include <fstream>
 #include <sstream>
+#include <fstream>
 
-using namespace boost::filesystem;
+using namespace std::filesystem;
```

Comment:
is not needed to include the filesystem library?

Reply:
this file include `UpdateFetcher.h` which imports `<filesystem>`